### ISSUE TITLE
High resolution images are used

### ISSUE THREAD

**Issue by schelv** [inaturalist/SeekReactNative#264]
Hi!

I like the app a lot.
Only it is quite slow.
For example when I go to "my observations" and select an observation it takes quite a while to load.

I suspect that the example images for a species are loaded in their original resolution. 
Because when I turn wifi off 8MB of data is used to load 5 images. 
This seems unnecessary.. maybe the app can become faster if lower resolution images are used.

**Comment by albullington** [comment]
Hi @schelv, thanks for expressing your concern about this. We are not using high resolution images for the My Observations list; instead, we're using small thumbnails.

What device are you using, and how many observations do you have? These are two factors that could be impacting how slowly the Observations list loads.

**Comment by schelv** [comment]
That's not what I'm trying to say.
Let me try again.

The My Observations list is shown.
At this point Android can display this months cellular data usage (up untill now) of the Seek app (for example 65.23MB).
In the observation list I click on an observation for example "Felis catus" 
The page is loaded, with in the upper part of the screen a picture shown of a cat.
If I swipe this image left or right a new image (of a cat) is shown.
Each time this is done, and a new image appears, the cellular data usage of the Seek app as monitored by the Android system goes up by up to 3MB.

Is it more clear now?

**Comment by albullington** [comment]
Yes, it sounds like you are saying the Species Detail screen is loading slowly and potentially causing some unnecessary data usage, which you notice when you scroll through images at the top of the screen.

Which device, Android OS, and version of Seek are you using? We've been working on performance improvements for that screen so it would be helpful to understand whether you're on the latest version.

**Comment by albullington** [comment]
We'll be using smaller sized versions of the Species Photos in Seek v2.10.5 -- thanks for bringing this to our attention.
